static/app/components/replays/replayCountContext.tsx

```diff
@@ -18,4 +23,45 @@ import type useReplaysCount from 'sentry/components/replays/useReplaysCount';
  * const count = useContext(ReplayCountContext)[groupId];
  * ```
  */
-export default createContext<ReturnType<typeof useReplaysCount>>({});
+const ReplayCountContext = createContext<ReturnType<typeof useReplaysCount>>({});
+
+export default ReplayCountContext;
```

Comment:
nit: can we make this a named export so we're not mixing default + named?

(I'd be in favor of adding a lint rule for this too)

Reply:
I think i'll probably pull out the named export into it's own file tbh. 

I didn't start out with that extra thing, but it's got enough imports and utility that i can figure out a unique name for it